src/components/application_manager/test/commands/mobile/change_registration_test.cc

```diff
@@ -475,6 +559,64 @@ TEST_F(ChangeRegistrationRequestTest, Run_IsLanguageSupportedByVrTrue_SUCCESS) {
   command_->Run();
 }
 
+TEST_F(ChangeRegistrationRequestTest,
+       Run_IsLanguageSupportedByTTSLanguagesEmpty_SUCCESS) {
+  CreateDefaultMessage(message_);
+  (*message_)[am::strings::msg_params][am::strings::language] = 1;
```

Comment:
@okozlovlux, why is `language` is `1`? As far as I know, languages have their own enum. Please check everywhere